src/staging/pg_conversion.py

```diff
@@ -54,14 +55,14 @@ def pg_to_pg_mapper(
     filtered_df[target_col] = filtered_df[pg_column].map(map_dict)
     filtered_df[target_col] = filtered_df[target_col].astype("category")
 
-    df.loc[
-        filtered_df.index,
-        f"{target_col}",
-    ] = filtered_df[target_col]
+    # df.loc[
+    #     filtered_df.index,
+    #     f"{target_col}",
+    # ] = filtered_df[target_col]
```

Comment:
shall we remove these commented out rows?

Reply:
Yes we can. I removed it when you and Jen were looking at it, then reincluded it, and then changed my mind again. Do you agree that it's right to have this removed? If so I will remove this commented out code